### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by XD-chill-bro**
also I added a "clear console" feature, so when the user presses enter the console will clear leaving only what is needed to read.

### CODE REVIEW

File: include/select.h [hunk 1]
```diff
@@ -0,0 +1,218 @@
+#include <iostream>
+#include <chrono>
+#include <unistd.h>
+
+int defaultScreenTime = 1200000;
+int userTimeInput;
+int userSelection[6] = {600, 1200, 1800, 2400, 3000, 3600};
+int restTime = 20;
+
+class select
```
Comment (matztron) [inline include/select.h]: Class name is confusing and too specific. I suggest making a "User-class" in which we store preferences and settings.

File: include/select.h [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+ 
+         sleep(restTime);
+         std::cout << "--------------------------------------------------------" << std::endl;
+
+         //Beep
+         std::cout << '\a';
+         std::cout << "Continue working :)" << std::endl;
+        }
+    }
+     else{
```
Comment (matztron) [inline include/select.h]: Empty else

File: include/select.h [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+    std::cout << "Press 1 for a reminder every 10 minutes\n";
+    std::cout << "Press 2 for a reminder every 20 minutes\n";
+    std::cout << "Press 3 for a reminder every 30 minutes\n";
+    std::cout << "Press 4 for a reminder every 40 minutes\n";
+    std::cout << "Press 5 for a reminder every 50 minutes\n";
+    std::cout << "Press 6 for a reminder every hour\n\n >";
+
+    std::cin >> userTimeInput;
+     
+    if (userTimeInput == 1)
```
Comment (matztron) [inline include/select.h]: Code duplication. I suggest making a function for that.

File: include/select.h (hunk 1 above)
Comment (matztron) [reply include/select.h]: Plus: This repo is for the 20-20-20 rule. I appreciate that you made this changeable. However 20min and 20sec should be the default values.

**Comment by matztron** [comment]
I really appreciate you helping out! :)

**Comment by XD-chill-bro** [comment]
> I really appreciate you helping out! :)

No problem bro

**Pull request merged**